packages/react-app/src/pages/manage.tsx

```diff
@@ -49,41 +55,48 @@ const Manage = () => {
         <Button
           sx={{
             textTransform: 'capitalize',
-            backgroundColor: activeTab === 'active' ? '#45507C' : 'transparent',
-            color: activeTab === 'active' ? 'white' : '#45507C',
+            backgroundColor: activeTab === Tabs.Active ? '#45507C' : 'transparent',
+            color: activeTab === Tabs.Active ? 'white' : '#45507C',
             ":hover": {
-              bgcolor: activeTab === 'active' ? '#45507C' : 'transparent',
-              color: activeTab === 'active' ? 'white' : '#45507C',
+              bgcolor: activeTab === Tabs.Active ? '#45507C' : 'transparent',
+              color: activeTab === Tabs.Active ? 'white' : '#45507C',
             }
           }}
-          variant={activeTab === 'active' ? "contained" : "text"}
-          onClick={() => { setActiveTab('active') }}
+          variant={activeTab === Tabs.Active ? "contained" : "text"}
+          onClick={() => { setActiveTab(Tabs.Active) }}
         >
           Active
         </Button>
         <Button
           sx={{
             textTransform: 'capitalize',
             ml: '1rem',
-            backgroundColor: activeTab === 'archived' ? '#45507C' : 'transparent',
-            color: activeTab === 'archived' ? 'white' : '#45507C',
+            backgroundColor: activeTab === Tabs.Archived ? '#45507C' : 'transparent',
+            color: activeTab === Tabs.Archived ? 'white' : '#45507C',
             ":hover": {
-              bgcolor: activeTab === 'archived' ? '#45507C' : 'transparent',
-              color: activeTab === 'archived' ? 'white' : '#45507C',
+              bgcolor: activeTab === Tabs.Archived ? '#45507C' : 'transparent',
+              color: activeTab === Tabs.Archived ? 'white' : '#45507C',
```

Comment:
maybe we can create const variables with the needed values to avoid repeating the same ternary several times